### PR TITLE
Create Release 0.2.0

### PR DESCRIPTION
**Pull request by robertjahn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,13 +101,24 @@ You should get a HTTP 200 response code and see the notification in either Slack
 1. Run these commands after adjusting the repo, image name, and tag in this example
     ```
     # build image
-    docker build -t robjahn/notification-service:0.1.0 .
+    docker build -t keptnexamples/notification-service:0.2.0 .
```
Comment (beeme1mr) [inline README.md]: Can we use a differ org name? Something like "keptn-contrib" or "keptn-extensions".

Comment (dirkwall) [reply README.md]: "keptncontrib" makes sense, then it's in line with the GitHub organization names. Last time I checked, hyphens were not allowed in docker hub organizations.

Comment (robertjahn) [reply README.md]: @dirkwall - happy to use a new "keptncontrib" org.  Just let me know when available and I will adjust the travis scripts.

Comment (robertjahn) [reply README.md]: I worked with Dirk and have adjusted to push images to this new org. 
  https://cloud.docker.com/u/keptncontrib/repository/docker/keptncontrib/notification-service

File: keptn-notifications.postman_collection.json [hunk 2]
```diff
@@ -6,20 +6,25 @@
...
-			"name": "new-artifact",
+			"name": "configuration-changed - Image 100",
 			"request": {
 				"method": "POST",
 				"header": [
 					{
 						"key": "Content-Type",
 						"name": "Content-Type",
-						"value": "application/cloudevents+json",
+						"value": "application/json",
```
Comment (beeme1mr) [inline keptn-notifications.postman_collection.json]: Are cloud events not being sent anymore?

File: package.json [hunk 3]
```diff
@@ -46,7 +46,7 @@
     "nodemon": "^1.18.9",
     "prettier": "^1.15.3",
     "supertest": "^3.4.1",
-    "ts-jest": "24.0.2",
+    "ts-jest": "23.10.0",
```
Comment (beeme1mr) [inline package.json]: Why was this downgraded?

File: src/@typings/cloud-events.ts [hunk 4]
```diff
@@ -90,21 +78,22 @@ export type EvaluationDone = Base<{
 export type Problem = Base<{
   State: string;
   ProblemID: string;
-  PID: string;
   ProblemTitle: string;
-  // { ProblemDetailsJSON }
-  ProblemDetails: unknown;
-  // { ImpactedEntities }
+  ProblemDetails: {
+    ProblemDetailsJSON: string;
```
Comment (beeme1mr) [inline src/@typings/cloud-events.ts]: Is this really a string type?

File: src/events/notification/configuration-changed.ts [hunk 5]
```diff
@@ -19,24 +19,43 @@ export class ConfigurationChangedNotification extends Notification {
...
+      }
       this.defaultNotification.facts.push({
-        name: 'Test Strategy',
-        value: configurationChanged.data.teststrategy
+        name: 'Action',
+        value: canaryAction
       })
     }
 
+    /*
```
Comment (beeme1mr) [inline src/events/notification/configuration-changed.ts]: Remove commented code

File: src/events/notification/problem.ts [hunk 6]
```diff
@@ -12,21 +12,25 @@ export class ProblemNotification extends Notification {
...
-          value: problem.data.PID
-        },
         {
           name: 'Problem ID',
           value: problem.data.ProblemID
         },
         { 
-          name: 'State',
-          value: problem.data.State
+          name: 'ImpactedEntity',
```
Comment (beeme1mr) [inline src/events/notification/problem.ts]: Change to "Impacted Entity"

**Comment by christian-kreuzberger-dtx** [comment]
I'm dismissing this PR. We will continue work in develop/master branch and a new release will be made for Keptn 0.6.0 compatibility.

**Pull request closed without merging**